content/2.0.html.haml

```diff
@@ -125,13 +139,22 @@ title: "Jenkins 2.0"
 
       For the more adventurous users, the Jenkins 2.0 initial setup process
       also allows you to pick and choose exactly the plugins you want to meet
-      your specific needs.
+      your specific needs. The list of plugins is hand-picked by Jenkins experts
+      from our popular plugins to ensure the best getting-started experience for
+      new users.
 
     %a{:href => '/images/getting-started-setup.png', :target => '_blank'}
       %img{:src => '/images/getting-started-setup.png', :title => 'Getting started with Jenkins 2.0',
           :width => 550}/
 
     %p
+      Additionally,
+      %em
+        starting in an upcoming alpha release
+      , the initial setup will be protected so that your Jenkins is always safe from unauthorized access.
```

Comment:
@orrc Both. Mostly. https://github.com/jenkinsci/jenkins/pull/2042 is going through a few iterations so I didn't want to be too specific. The idea is that at no point in time Jenkins is accessible to anonymous users unless the legitimate admin configures it like that.